SimpleAudioPlayer/Teddy.MvvmCross.Plugins.SimpleAudioPlayer/MvxSimpleAudioPlayerExtensions.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Teddy.MvvmCross.Plugins.SimpleAudioPlayer
+{
+    public static class SimpleAudioPlayerExtensions
+    {
+        public static void SetUpLooping(this IMvxSimpleAudioPlayer player)
+        {
+            var playbackHandler = new EventHandler(delegate (Object o, EventArgs a)
```

Comment:
instead of new EventHandler() on SetUpLooping() call and use a dictionary, better to define a private method and += and -= the private method to player.Completion event.